sylvia-derive/src/message.rs

```diff
@@ -212,6 +212,17 @@ impl<'a> EnumMessage<'a> {
 
         let resp_type = match associated_exec {
             Some(exec) if !custom.has_msg() => exec,
+            None if !custom.has_msg() => {
+                // This will show only on +nightly according to:
+                // https://docs.rs/proc-macro-error/latest/proc_macro_error/macro.emit_warning.html
+                emit_warning!(
+                    source.span(), "The trait is missing the custom message definition, and returning any chain-specific custom messages on its implementation will not be possible.";
+                    note = "Consider adding the `type ExecC;` associated message in the trait definition, and then use it in the execution responses return type: `Result<Response<Self::ExecC>, _>`";
+                    note = "If it is intended and you don't want to allow executing custom messages from this trait, add the `#[sv::custom(cosmwasm_std::Empty)]` attribute on the trait.";
```

Comment:
Slight typo in second note:

```suggestion
                    note = "If it is intended and you don't want to allow executing custom messages from this trait, add the `#[sv::custom(msg=cosmwasm_std::Empty)]` attribute on the trait.";
```

I know it is what was in task, but it is incorrect.